examples/angular/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "angular-example",
+  "version": "0.0.0",
+  "scripts": {
```

Comment:
Please add the scripts from TypeScript example: https://github.com/contentful/apps/blob/be8304afc8cc52b89cfded06af6e405f7f004515/examples/typescript/package.json#L21-L23